pkg/clusteragent/admission/mutate/autoinstrumentation/language_versions.go

```diff
@@ -134,9 +134,9 @@ type libInfo struct {
 	image   string
 }
 
-func (i libInfo) podMutator(ics []containerMutator, ms []podMutator) podMutator {
+func (i libInfo) podMutator(v version, ics []containerMutator, ms []podMutator) podMutator {
```

Comment:
That's right, all of v1 is expected to go away and we can decide if we want to either error out or just say "warning: unsupported" or ignore the choice regardless.

Reply:
The implementation here is that `libInfo` is the factory function for `libRequirements` now-- I think I basically did what you were saying but didn't change info into a struct.

Agree it would have been cleaner to have a libInfov1 and v2 though.